### PR TITLE
Tweak script to generate MLflow environment tarball

### PR DESCRIPTION
**Pull request by adriangonz**
**What this PR does / why we need it**:
Tweak the script which generates the MLflow demo artifact so that it also creates an `environment.tar.gz` tarball.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **adriangonz**
You can assign the PR to them by writing `/assign @adriangonz` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: servers/mlflowserver/models/elasticnet_wine/Makefile
Comment (RafalSkolasinski) [inline servers/mlflowserver/models/elasticnet_wine/Makefile:21]: instead of `-n elasticnet-wine-env` could we use environment in local folder?

File: servers/mlflowserver/models/elasticnet_wine/Makefile
Comment (RafalSkolasinski) [inline servers/mlflowserver/models/elasticnet_wine/Makefile:22]:
```suggestion
		.env/bin/conda-pack -n elasticnet-wine-env -f -o environment.tar.gz
```

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
models | ac1305d188ed8bbec31b16f503d6d317f4294943 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/donio-seldon-core-pr-3794-models-pvj8q) | `/test models`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: servers/mlflowserver/models/elasticnet_wine/Makefile
Comment (adriangonz) [reply servers/mlflowserver/models/elasticnet_wine/Makefile:21]: Do you mean using `-p ./env/elasticnet-wine-env`? That's a great point. I'll update it.

**Comment by seldondev** [comment]
@adriangonz: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
models | 39c05d9aea7998029d43266723b58f936b4022c3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/donio-seldon-core-pr-3794-models-tvfdn) | `/test models`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
Seems to work fine
```
rclone ls gs://seldon-models/v1.12.0-dev/
      350 mlflow/elasticnet_wine/MLmodel
      143 mlflow/elasticnet_wine/conda.yaml
234734119 mlflow/elasticnet_wine/environment.tar.gz
      185 mlflow/elasticnet_wine/metadata.yaml
```

**Comment by RafalSkolasinski** [comment]
image

**Review by RafalSkolasinski — APPROVED** [review]

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[RafalSkolasinski](https://github.com/SeldonIO/seldon-core/pull/3794#pullrequestreview-826333711 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [RafalSkolasinski]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
